test/framework/workload_flux.go

```diff
@@ -0,0 +1,98 @@
+package framework
```

Comment:
I don't think we need a new file for this, this is still part of the workload test code. If you see yourself making the original file too big, that's a good sign your entity has grown too big.